src/content/en/fundamentals/instant-and-offline/get-started/textcontent-3.md

```diff
@@ -0,0 +1,276 @@
+
+Here, we ran the test on GIDNetwork's own root URL -- interestingly, it did not appear 
+to be compressed.
+
+![GIDNetwork](images/image_303.png)
+
+Gzip can further compress already-compressed files, and that's a valid approach. In fact, 
+to get the greatest compression ratio for text-based resources, first minify them individually 
+prior to deployment and then compress them at delivery via a Gzip-enabled server.
+
+## Reduce Library Use
+
+Popular CSS and JavaScript libraries do their best to minify and compress their download files, 
+but generally they're still pretty serious bandwidth consumers. jQuery, for example -- depending 
+on the version and the compression algorithms applied -- might range from 28k to over 250k. 
+If you need most of a given library's features, fine; but if you only need one or two specific 
+things, you can save a lot of download time by replacing those features with single-use 
+functions or CSS rules.
+
+For example, a website might use jQuery's handy `slideToggle` feature to expand and collapse 
+a div, like this.
+
+`<a href="javascript:void(0)" onclick="$('#more').slideToggle()">More...</a>`
+
+Yes, it works great and it's easy to code, but jQuery is a lot of download overhead for one text 
+effect. You might consider swapping out that huge library for a simpler -- and far smaller -- 
+single-purpose function, like this 
+(source: [W3Schools](https://www.w3schools.com/howto/howto_js_toggle_hide_show.asp)).
+
+```
+function myFunction() {
+  var x = document.getElementById('myDIV');
+  if (x.style.display === 'none') {
+    x.style.display = 'block';
+  } else {
+    x.style.display = 'none';
+  }
+}
+```
+
+You could even go with pure CSS using a transition, like this nifty 475-character solution 
+(source: [CodePen/Paul O'Brien](https://codepen.io/paulobrien/pen/tpmAi)).
+
+```
+label {
+    display:block;
+    margin:20px 0 0;
+    border-bottom:1px solid green;
+}
+label:hover { text-decoration:underline }
+input {
+    position:absolute;
+    left:-999em
+}
+.hide {
+    width:50%;
+    border:1px solid #000;
+    background:red;
+    max-height:99em;
+    opacity:1;
+    height:auto;
+    overflow:hidden;
+    transition:opacity 1.5s linear, max-height 1.5s linear;
+}
+.hide p {
+    padding:10px;
+    margin:0
+}
+input[type=checkbox]:checked + div {
+    opacity:0;
+    max-height:0;
+    border:none;
+}
+.follow{border-top:1px solid blue;margin:0}
+```
+
+The point is, if you don't need an entire 250k library, don't download it. Instead, find 
+and use small, single-purpose routines that do only what you need. (And don't forget to 
+minify them!)
+
+You can find an interesting and to-the-point take on alternatives to jQuery code at 
+[You Might Not Need jQuery](http://youmightnotneedjquery.com/), which explores how the 
+modern web has evolved to provide many of the same capabilities for which we might 
+have previously used jQuery.
+
+## Prioritize Text Content
```

Comment:
I would drop this "Prioritize Text Content" section. The notes around CSS + DOM positioning and impact on loading/rendering are missing a lot of nuance and imo, aren't as central to the key theme of loading you're trying to get across here. If you'd like to keep this in place, happy to provide some extra notes but I think this doc could work just as well without this part.